extensions/positron-run-app/src/api.ts

```diff
@@ -353,18 +353,27 @@ async function previewUrlInExecutionOutput(execution: vscode.TerminalShellExecut
 		return false;
 	}
 
-	// Convert the url to an external URI.
+	// Example: http://localhost:8500
 	const localBaseUri = vscode.Uri.parse(url.toString());
+
+	// Example: http://localhost:8500/url/path or http://localhost:8500
 	const localUri = urlPath ?
 		vscode.Uri.joinPath(localBaseUri, urlPath) : localBaseUri;
 
-	log.debug(`Viewing app at local uri: ${localUri} with external uri ${proxyInfo.externalUri.toString()}`);
+	// Determine the target origin based on whether the server is running in PWB.
+	const runningInPWB = process.env.RS_SERVER_URL ? true : false;
```

Comment:
Maybe we want this check in a command in case we end up using it in a bunch of places?

Reply:
That seems reasonable.

Reply:
I'm going to defer this to a future task.